### PR TITLE
 🐣 Add skeleton for `amp-video-service`

### PR DESCRIPTION
**Pull request by alanorozco**
The upcoming refactor PR is growing quite large, so I decided to split it up.

No validation rules as it's invalid to include the extension explicitly.

Added a simple `gulpfile` step to build `amp-video-service` optionally in case an extension requires it.

### CODE REVIEW

File: extensions/amp-video-service/amp-video-service.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<!--
```
Comment (aghassemi) [inline extensions/amp-video-service/amp-video-service.md]: I don't think this needs a public `.md` file, would be confusing for it to show up on ampproject.org as it is not user facing. I suggest moving any important documentation to the top of the `.js` file

File: extensions/amp-video-service/0.1/amp-video-service.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+ * @visibleForTesting
+ */
+export class VideoService {
+  /** @param {!../../../src/service/ampdoc-impl.AmpDoc} ampdoc */
+  constructor(ampdoc) {
+    dev().error('Unimplemented');
+  }
+}
+
+AMP.extension('amp-video-manager', 0.1, function(AMP) {
```
Comment (aghassemi) [inline extensions/amp-video-service/0.1/amp-video-service.js]: `amp-video-service`

File: extensions/amp-video-service/amp-video-service.md (hunk 1 above)
Comment (alanorozco) [reply extensions/amp-video-service/amp-video-service.md]: Good call, done.

File: extensions/amp-video-service/0.1/amp-video-service.js (hunk 2 above)
Comment (alanorozco) [reply extensions/amp-video-service/0.1/amp-video-service.js]: whoops, fixed.

**Comment by alanorozco** [comment]
@aghassemi also added `video-service` experiment def post-review.

**Pull request merged**